### PR TITLE
Fix migration to a kubernetes container

### PR DESCRIPTION
**Pull request by cbosdo**
In order to share more code with the install, some code has been shuffled around. This PR requires helm chart changes submitted in PR https://github.com/uyuni-project/uyuni/pull/7518

### CODE REVIEW

File: uyuniadm/shared/templates/migrateScriptTemplate.go [hunk 1]
```diff
@@ -7,19 +7,40 @@ import (
...
+  rsync -e "ssh -A " --rsync-path='sudo rsync' -avz {{ .SourceFqdn }}:$folder/ $folder;
 done;
 rm -f /srv/www/htdocs/pub/RHN-ORG-TRUSTED-SSL-CERT;
 ln -s /etc/pki/trust/anchors/LOCAL-RHN-ORG-TRUSTED-SSL-CERT /srv/www/htdocs/pub/RHN-ORG-TRUSTED-SSL-CERT;
 
+echo "Extracting time zone..."
 ssh {{ .SourceFqdn }} timedatectl show -p Timezone >/var/lib/uyuni-tools/data
 
 {{ if .Kubernetes }}
+echo "Altering configuration for kubernete..."
```
Comment (mbussolotto) [inline uyuniadm/shared/templates/migrateScriptTemplate.go:22]: tiny little typo

File: uyuniadm/shared/templates/migrateScriptTemplate.go [hunk 2]
```diff
@@ -7,19 +7,40 @@ import (
...
 echo 'server.no_ssl = 1' >> /etc/rhn/rhn.conf;
-sed 's/address=[^:]*:/address=uyuni:/' -i /etc/rhn/taskomatic.conf;
-sed 's/address=[^:]*:/address=uyuni:/' -i /etc/sysconfig/tomcat;
+sed 's/address=[^:]*:/address=*:/' -i /etc/rhn/taskomatic.conf;
+
+if test ! -f /etc/tomcat/conf.d/remote_debug.conf -a -f /etc/sysconfig/tomcat; then
+  cp /etc/sysconfig/tomcat /etc/tomcat/conf.d/remote_debug.conf
```
Comment (mbussolotto) [inline uyuniadm/shared/templates/migrateScriptTemplate.go:27]: maybe better to `mv` it?

**Review by mbussolotto — APPROVED** [review]
such a complex PR :)  I just noticed two little things. From me you have green light, I think the best would be test it ;)

**Review by rjmateus — APPROVED** [review]
LGTM, lets merge it

**Pull request merged**